src/content/ui-api/kubectl-gs/template-app.md

```diff
@@ -47,41 +48,30 @@ produces the following output:
 apiVersion: application.giantswarm.io/v1alpha1
 kind: App
 metadata:
-  creationTimestamp: null
-  labels:
-    app-operator.giantswarm.io/version: 1.0.0
-  name: my-app
-  namespace: default
+  name: keda
+  namespace: 2hr7z
 spec:
-  catalog: pipo-catalog
-  config:
-    configMap:
-      name: 2hr7z-cluster-values
-      namespace: 2hr7z
-    secret:
-      name: ""
-      namespace: ""
+  catalog: giantswarm-playground
   kubeConfig:
-    context:
-      name: 2hr7z-kubeconfig
     inCluster: false
-    secret:
-      name: 2hr7z-kubeconfig
-      namespace: 2hr7z
-  name: my-app
-  namespace: 2hr7z
-  userConfig:
-    configMap:
-      name: ""
-      namespace: ""
-    secret:
-      name: ""
-      namespace: ""
+  name: keda
+  namespace: default
   version: 0.1.0
-status:
-  appVersion: ""
-  release:
-    lastDeployed: "0001-01-01T00:00:00Z"
-    status: ""
-  version: ""
 ```
+
+It also supports the following optional flags:
+
+- `--defaulting-enabled`: Don't template fields that will be defaulted. (default true)
+- `--user-configmap`: Path to the user values configmap YAML file.
+- `--user-secret`: Path to the user secrets YAML file.
+
+Only required fields are templated. Other fields like the path to the kubeconfig secret are set by the
+[defaulting webhook]({{< relref "/app-platform/defaulting-validation" >}}).
+
+This is enabled for Giant Swarm releases.
+
+- AWS >= v14.0.0
+- Azure >= v13.1.0
+- KVM >= v13.1.0
+
+For older releases you can set the `--defaulting-enabled` flag to false.
```

Comment:
Ah, here it is. As mentioned, I think I'd like this detail a bit further up near `Only required fields are templated.`.